Resources/views/CRUD/base_list.html.twig

```diff
@@ -176,16 +176,19 @@ file that was distributed with this source code.
                                 </div>
 
 
+                                {# NEXT_MAJOR : remove this assignment #}
+                                {% set export_formats = export_formats|default(admin.getExportFormats) %}
```

Comment:
Just to make sure, `admin.getExportFormats` will work the same as `admin.getExportFormats()`?

AFAIK you can do something like `admin.exportFormats` at twig will pick the getter.

If this works also, lets merge this